setup.cfg

```diff
@@ -0,0 +1,43 @@
+[bumpversion]
+current_version = 0.1.0
+commit = True
+tag = True
+
+[bumpversion:file:setup.py]
+search = version='{current_version}'
+replace = version='{new_version}'
+
+[bumpversion:file:ravenpy/__init__.py]
+search = __version__ = '{current_version}'
+replace = __version__ = '{new_version}'
```

Comment:
need to add `.cruft.json` to list of files to bump version as well, like here https://github.com/Ouranosinc/raven/blob/58e8978375565fa095e0ec8ae1bf81dc25741f94/setup.cfg#L31-L33